editor/property_editor.h

```diff
@@ -304,6 +313,12 @@ class PropertyEditor : public Control {
 	void set_property_selectable(bool p_selectable);
 
 	void set_use_folding(bool p_enable);
+
```

Comment:
A line above you have a set_use_folding() function, just call that if folding is disabled and the editor will not fold by default and will not save any folded/unfolded state. I think it's easier than the two functions below.

Reply:
@reduz done, check.